addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -620,6 +623,7 @@ HtmlField.props = {
     isTranslatable: { type: Boolean, optional: true },
     placeholder: { type: String, optional: true },
     fieldName: { type: String, optional: true },
+    forceCodeView: {type: Boolean, optional: true},
```

Comment:
The commit message is talking about a new `xml` option but I can't find it. I feel like `forceCodeView` here might serve the same purpose from a functional point of view. I think it is weird to have a `forceCodeView` option because you can instantiate the editor with `forceCodeView: true` and `codeview: false` and it's just weird IMO. @Goaman What do you think ? Would we prefer something like `type` to which you could give `'xml'` instead of the default `'html'` ? In any case, it's the _html_ editor so it's kind of weird anyway...

Reply:
> XML option
Yeah I meant it's an option, which we set in the XML here. I'll rephrase the commit message.

Well, would be nice to have some way to instanciate the 'xml editor' since it exists. I can always just set it to be clear text, but 'xml editor' just seems to make more sense.

Reply:
ping @Goaman